scripts/homebrew/template.rb

```diff
@@ -0,0 +1,58 @@
+class Iota < Formula
+    desc "Bringing the real world to Web3 with a scalable, decentralized and programmable DLT infrastructure"
+    homepage "https://www.iota.org"
+    license "Apache-2.0"
+
+    version "{{version}}"
+    checksums = {
+        "macos-arm64" => "{{macos-arm64-checksum}}",
+        "linux-x86_64" => "{{linux-x86_64-checksum}}",
+        "source" => "{{source-checksum}}"
+    }
+    @@arch = "source"
+
+    on_macos do
+        on_arm do
+            @@arch = "macos-arm64"
+        end
+    end
+
+    on_linux do
+        on_intel do
+            @@arch = "linux-x86_64"
+        end
+        depends_on "llvm" => :build
```

Comment:
Should not be installed if a binary exists

Reply:
done